language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/CommandSubmissionClient.java

```diff
@@ -15,6 +16,15 @@
 /** An RxJava version of {@link com.daml.ledger.api.v1.CommandSubmissionServiceGrpc} */
 public interface CommandSubmissionClient {
 
+  Single<Empty> submit(CommandsSubmission params);
```

Comment:
```suggestion
  Single<Empty> submit(CommandsSubmission commandsSubmission);
```

Super nitpicky, I know :-)